queries.sql

```diff
@@ -1,3 +1,15 @@
-/*Queries that provide answers to the questions from all projects.*/
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
```

Comment:
- [ ] Here we are trying to select the name and not selecting all records, kindly fix this issue.
![image](https://user-images.githubusercontent.com/81932478/212674066-b1dc8b1c-0009-4a68-b496-5aff0f1e975f.png)